### PR TITLE
Fab deploy improvement

### PR DESCRIPTION
**Pull request by maxsocl**
@VladimirFilonov 

There are proposed improvements for `fab db` tasks.
Fabric tasks allows to pass host, port, user, password, db_name params, uses temp pgpass file, etc.

According to #102

### CODE REVIEW

File: fabfile/db/__init__.py [hunk 1]
```diff
@@ -97,6 +145,23 @@ def run(self, action, suffix=None):
...
+            self.db_cfg['NAME'] = self.name
+        if self.user:
+            self.db_cfg['USER'] = self.user
+        if self.password:
+            self.db_cfg['PASSWORD'] = self.password
+
+        if self.host:
+            self.db_cfg['HOST'] = self.host
+        elif not self.db_cfg['HOST']:
+            self.db_cfg['HOST'] = 'localhost'
```
Comment (sengottuvelk) [inline fabfile/db/__init__.py]: I would write this code block into something like
```python
self.db_cfg['HOST'] = self.host or 'localhost'
```

File: fabfile/db/__init__.py [hunk 2]
```diff
@@ -97,6 +145,23 @@ def run(self, action, suffix=None):
...
+
+        if self.port:
+            self.db_cfg['PORT'] = self.port
+        elif not self.db_cfg['PORT']:
+            self.db_cfg['PORT'] = '5432'
+
```
Comment (sengottuvelk) [inline fabfile/db/__init__.py]: Same as above,
```python
self.db_cfg['PORT'] = self.port or '5432'
```

**Comment by sengottuvelk** [comment]
Otherwise, LGTM. Merge when ready.

File: fabfile/db/__init__.py (hunk 2 above)
Comment (maxsocl) [inline fabfile/db/__init__.py]: > Same as above,
> 
> self.db_cfg['PORT'] = self.port or self.db_cfg['PORT'] or '5432'

Refactored

File: fabfile/db/__init__.py (hunk 1 above)
Comment (maxsocl) [inline fabfile/db/__init__.py]: > I would write this code block into something like
> 
> self.db_cfg['HOST'] = self.host or self.db_cfg['HOST'] or 'localhost'

Refactored

**Pull request merged**